build.sbt

```
@@ -44,6 +44,10 @@ lazy val commonDependencies = Seq(
   silencerPlugin,
   silencerLib
 )
+lazy val cliDependencies = Seq(
+  commonDependencies,
+  scopt
+)
```

Сейчас у тебя получилось:
```scala
lazy val cliDependencies = Seq(
  Seq(
    scalastic,
    scalatest,
    silencerPlugin,
    silencerLib
  ),
  scopt
)
```
А нужно:
```scala
lazy val cliDependencies = Seq(
  scalastic,
  scalatest,
  silencerPlugin,
  silencerLib,
  scopt
)
```
```suggestion
lazy val cliDependencies = commonDependencies :+ scoptt
```